pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -483,6 +483,7 @@ const (
 	RoleBindingV1GeneratorName              = "rolebinding.rbac.authorization.k8s.io/v1alpha1"
 	ClusterV1Beta1GeneratorName             = "cluster/v1beta1"
 	PodDisruptionBudgetV1GeneratorName      = "poddisruptionbudget/v1beta1"
+	PodDisruptionBudgetV2GeneratorName      = "poddisruptionbudget/v1beta1/ge1.7"
```

Comment:
@kubernetes/sig-cli-api-reviews we need to come up with a standarized naming form for generators that build on top of exisitng generators.